package.json

```diff
@@ -20,6 +20,9 @@
     "build:core:css": "sass packages/core/style:packages/core/dist --no-source-map -s compressed",
     "build:core:esm": "tsc -p packages/core/tsconfig.json",
     "build:react:esm": "tsc -p packages/react/tsconfig.json",
+    "preprototype": "shx rm -rf packages/*/prototype",
+    "prototype": "concurrently yarn:prototype:*",
+    "prototype:react": "parcel build packages/react/src/index.ts --out-dir packages/react/prototype --out-file react.js --global Castor --no-minify --no-source-maps",
```

Comment:
🤔  why no source maps? Should help troubleshoot any potential error, if one is thrown.